app/models/payment/PaymentRecord.scala

```diff
@@ -40,49 +39,57 @@ case class PaymentRecord(reference: String,
 }
 
 object PaymentRecord {
+  val createdOnFormatter: DateTimeFormatter = DateTimeFormatter.ISO_DATE_TIME
+
+  val formatFullDtf: DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM yyyy")
 
   private[payment] object DateFormatting {
     def formatFull(date: LocalDate)(implicit messages: Messages): String =
-      createDateFormatForPattern("d MMMM yyyy", messages).format(date.toDate)
+    createDateFormatForPattern("d MMMM yyyy", messages).format(date)
+
 
-    private def createDateFormatForPattern(pattern: String, messages: Messages): SimpleDateFormat = {
+    private def createDateFormatForPattern(pattern: String, messages: Messages): DateTimeFormatter = {
       val langCode = messages.lang.code
-      val uk = TimeZone.getTimeZone("Europe/London")
-      val validLang: Boolean = ULocale.getAvailableLocales.contains(new ULocale(langCode))
-      val locale: ULocale = if (validLang) new ULocale(langCode) else ULocale.getDefault
-      val sdf = new SimpleDateFormat(pattern, locale)
-      sdf.setTimeZone(uk)
+      val validLang: Boolean = Locale.getAvailableLocales.contains(new Locale(langCode))
+      val locale: Locale = if (validLang) new Locale(langCode) else Locale.getDefault
+      val sdf = DateTimeFormatter.ofPattern(pattern, locale)
       sdf
     }
   }
 
-  def from(paymentRecordData: VatPaymentRecord, currentDateTime: DateTime): Option[PaymentRecord] =
+  def from(paymentRecordData: VatPaymentRecord, currentDateTime: LocalDateTime): Option[PaymentRecord] = {
+
     if (paymentRecordData.isValid(currentDateTime) && paymentRecordData.isSuccessful) {
       Some(PaymentRecord(
         reference = paymentRecordData.reference,
         amountInPence = paymentRecordData.amountInPence,
-        createdOn = new DateTime(paymentRecordData.createdOn),
+        createdOn = LocalDateTime.parse(paymentRecordData.createdOn, createdOnFormatter),
```

Comment:
Note this will give us a simple date/time unlike Joda time's DateTime. Need to check this doesn't cause any problems further on.

Reply:
I think we are probably Ok. Internally, payment records are used by the SubPageController. Ultimately, this functionality displays the date/time using the formatter in payment record. The payments are also used by  the card connector which supplies BTA with data. This application has a similar instance of Payment record.